OpenRA.Mods.Common/Traits/World/ShroudRenderer.cs

```diff
@@ -299,7 +299,7 @@ void UpdateShroud(ProjectedCellRegion region)
 
 		void IRenderShroud.RenderShroud(WorldRenderer wr)
 		{
-			UpdateShroud(map.ProjectedCellBounds);
+			UpdateShroud(map.ProjectedCells);
```

Comment:
I could have sworn that we were only checking/updating the cells that are actually visible on screen this render. It is a waste of time to resolve anything else, because there is a strong chance they will be dirtied again before we next move the viewport to see them...

This is a bigger deal than my other comment because we probably need to keep `UpdateShroud` taking a `ProjectedCellRegion` in order to restore this behaviour!